src/Shouldly/ShouldStaticClasses/ShouldThrowExtensions.cs

```diff
@@ -8,6 +8,20 @@ namespace Shouldly
     [ShouldlyMethods]
     public static partial class Should
     {
+        /*** ShouldThrow(Action) ***/
+        public static TException ShouldThrow<TException>(this Action actual) where TException : Exception
```

Comment:
I think we should put these extensions in another class so we don't have `Should.ShouldThrow` and `Should.Throw` on the same class.

Maybe move it `ShouldThrowExensions` or something